### PR TITLE
Update economics

### PR DESCRIPTION
**Pull request by AdoAdoAdo**
add changes from development branch

### CODE REVIEW

File: integrationTests/testProcessorNode.go [hunk 1]
```diff
@@ -42,8 +42,9 @@ import (
 	"github.com/ElrondNetwork/elrond-go/process/smartContract/hooks"
 	"github.com/ElrondNetwork/elrond-go/process/transaction"
 	"github.com/ElrondNetwork/elrond-go/sharding"
-	vmcommon "github.com/ElrondNetwork/elrond-vm-common"
+	"github.com/ElrondNetwork/elrond-vm-common"
 	"github.com/pkg/errors"
+	factory2 "github.com/ElrondNetwork/elrond-go/data/state/factory"
```
Comment (sasurobert) [inline integrationTests/testProcessorNode.go]: delete factory2

File: process/smartContract/process_test.go [hunk 2]
```diff
@@ -14,6 +14,7 @@ import (
 	"github.com/ElrondNetwork/elrond-vm-common"
 	"github.com/pkg/errors"
 	"github.com/stretchr/testify/assert"
+	"github.com/ElrondNetwork/elrond-go/process/coordinator"
```
Comment (BeniaminDrasovean) [inline process/smartContract/process_test.go]: Sort imports.

File: process/coordinator/process_test.go [hunk 3]
```diff
@@ -2,29 +2,28 @@ package coordinator
...
-	"github.com/ElrondNetwork/elrond-go/process/factory"
 	"github.com/ElrondNetwork/elrond-go/process/factory/shard"
 	"github.com/ElrondNetwork/elrond-go/process/mock"
 	"github.com/ElrondNetwork/elrond-go/storage"
 	"github.com/ElrondNetwork/elrond-go/storage/memorydb"
 	"github.com/ElrondNetwork/elrond-go/storage/storageUnit"
 	"github.com/stretchr/testify/assert"
+	"github.com/ElrondNetwork/elrond-go/data"
+	"math/big"
+	"github.com/ElrondNetwork/elrond-go/core"
```
Comment (sasurobert) [inline process/coordinator/process_test.go]: sort imports

File: process/smartContract/process_test.go (hunk 2 above)
Comment (sasurobert) [inline process/smartContract/process_test.go]: goimports

File: process/unsigned/feeTxHandler.go [hunk 4]
```diff
@@ -13,10 +13,10 @@ import (
 )
 
 // MinGasPrice is the minimal gas price to be paid for any transaction
-var MinGasPrice = uint64(1)
+var MinGasPrice = uint64(0)
```
Comment (sasurobert) [inline process/unsigned/feeTxHandler.go]: add a TODO here to put back

File: consensus/broadcast/shardChainMessenger_test.go [hunk 5]
```diff
@@ -1,6 +1,7 @@
 package broadcast_test
 
 import (
+	"github.com/ElrondNetwork/elrond-go/process/factory"
```
[the remark this answers is not in the document]
Comment (AdoAdoAdo) [reply consensus/broadcast/shardChainMessenger_test.go]: done

File: integrationTests/testProcessorNode.go (hunk 1 above)
Comment (AdoAdoAdo) [reply integrationTests/testProcessorNode.go]: done

File: process/coordinator/process_test.go (hunk 3 above)
Comment (AdoAdoAdo) [reply process/coordinator/process_test.go]: done

File: process/smartContract/process_test.go (hunk 2 above)
Comment (AdoAdoAdo) [reply process/smartContract/process_test.go]: done

File: process/unsigned/feeTxHandler.go (hunk 4 above)
Comment (AdoAdoAdo) [reply process/unsigned/feeTxHandler.go]: done

File: integrationTests/testProcessorNode.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (AdoAdoAdo) [reply integrationTests/testProcessorNode.go]: done

File: process/coordinator/process_test.go [hunk 6]
```diff
@@ -2,29 +2,28 @@ package coordinator
...
 	"github.com/ElrondNetwork/elrond-go/dataRetriever"
 	"github.com/ElrondNetwork/elrond-go/process"
-	"github.com/ElrondNetwork/elrond-go/process/factory"
 	"github.com/ElrondNetwork/elrond-go/process/factory/shard"
 	"github.com/ElrondNetwork/elrond-go/process/mock"
 	"github.com/ElrondNetwork/elrond-go/storage"
 	"github.com/ElrondNetwork/elrond-go/storage/memorydb"
 	"github.com/ElrondNetwork/elrond-go/storage/storageUnit"
 	"github.com/stretchr/testify/assert"
+	"github.com/ElrondNetwork/elrond-go/data"
```
[the remark this answers is not in the document]
Comment (AdoAdoAdo) [reply process/coordinator/process_test.go]: done

File: process/smartContract/process_test.go (hunk 2 above)
Comment (AdoAdoAdo) [reply process/smartContract/process_test.go]: done

**Pull request merged**